ts/src/bitget.ts

```diff
@@ -2297,7 +2297,7 @@ export default class bitget extends Exchange {
             const frozen = this.safeString (entry, 'frozen');
             const locked = this.safeString2 (entry, 'lock', 'locked');
             account['used'] = Precise.stringAdd (frozen, locked);
-            account['free'] = this.safeString (entry, 'available');
+            account['total'] = this.safeString (entry, 'available');
```

Comment:
this does not seem right
![image](https://github.com/ccxt/ccxt/assets/43336371/2a9d0c6d-9f0b-426b-acb6-32932bc6ebe1)
![image](https://github.com/ccxt/ccxt/assets/43336371/7be17156-1934-432e-b45b-a2e6625a170c)

Reply:
@carlosmiei which account did you use?

Reply:
@sc0Vu spot

Reply:
@carlosmiei that's weird, it works for me. I only have one spot order in my account. Do you have any other orders or positions?

<img width="809" alt="image" src="https://github.com/ccxt/ccxt/assets/10494397/8ba73d2f-f455-44bd-ade8-a53fc631c1f3">


<img width="351" alt="image" src="https://github.com/ccxt/ccxt/assets/10494397/190d4c13-1e69-4e36-a311-ba56c82f1056">